rules/manager_test.go

```diff
@@ -221,7 +221,7 @@ func TestStaleness(t *testing.T) {
 	testutil.Equals(t, want, samples)
 }
 
-// Convert a SeriesSet into a form useable with reflect.DeepEqual.
+// Convert a SeriesSet into a form usable with reflect.DeepEqual.
```

Comment:
IMHO This isn't a spelling mistake - both are acceptable - even in US English. :)

Reply:
@jamtur01 Removed.